economics/staking_protocol/main.tex

```diff
@@ -0,0 +1,355 @@
+\\\\
+The second implication of these results is that, in some cases, changes in the real yield is having the opposite effect as would be expected. The Staking Rewards data suggests that the collective staker reaction to lower real yields may be to increase their stake, and vice versa. This has 
+\\\\
+It's worth mentioning that the negative cross-correlations on display theoretically contain both the reaction of staking more in response to a lower real yield, and staking less in response to a higher real yield – but crucially, the latter necessarily occurs over with a time delay. It is not easy to disentangle these two dynamics within the results data. Nevertheless, we can make claims about specific data points – for example, let's look at the negative cross-correlation of -0.965 calculated for Cosmos, at a lag of 14 days (56 x 6 hour periods), in the 14 day lump, with Staking Rewards data. Given that Cosmos's unbonding delay is 21 days, this result is almost entirely explained by stakers depositing tokens, as opposed to withdrawing them. 
+\\\\
+More generally, a major unknown in these systems is the staker `reaction time'. Theoretically, a staker could respond to a changing real yield immediately, while another waits weeks for this change to evolve into a trend. Selecting a single lump size would introduce unacceptable experimenter bias into the results. Of course, 7, 14, 21, and 28 day lumps are not the full story, but they are a decent starting point. Lumps lower than 7 days seem unrealistic, since anecdotally, stakers are not known to change their stake size in response to hourly or daily changes to subsidies. The very weak correlations at the 7 day lump size demonstrate this. Lumps greater than 28 days run into sample size issues, where the total number of data points to run a Pearson correlation over drops to single digits.
+
+\subsubsection{Interpretation with respect to two-phase model}
+
+The notion that \textit{node operators can only be counted on for a reliable service if they are well compensated, or will quickly abandon the network} appears, so far, to be empirically unsubstantiated. None of the networks under examination, based on data from two independent sources, have experienced a shortfall of operators, nor are there any observable trends that correlate decreasing earnings with an increasing paucity of supply. The observable abundance of supply and lack of staker sensitivity to decreasing rewards appears to \textbf{derisk the two-phase model}, by undermining a possible objection – that relatively lower issuance in the early days of the network (see figure 1), will fail to attract or sustain a sufficient number of node operators. Remember that all the data in this study pertains to the very early months of each network's existence, so it is a reasonable model for the first few years of the NuCypher network.
```

Comment:
still... the emission is still among the highest among staking projects even with the two-phase model so it feels a bit awkward to answer a "low inflation" objection

Reply:
This is a really good point. I've rewritten the interpretation: 

Regardless of which of these underlying explanations one favours, it is reasonable to conclude that reserving the highest subsidies for the early years of the network is wholly unnecessary. The essence of the two-phase model is maintaining the sum of tokens to be minted over the network's lifetime, but flattening the minting schedule. Therefore the riskiest epoch of the two-phase model, relative to a generic model (i.e. where the issuance falls continuously), is the first few years – where all things equal, the issuance is lower than it would have been. The results demonstrate that, whether due to enthusiasm or deep pockets, service-providers are very unlikely to abandon the network early on. Given the over-arching objective of abundant and reliable service provision, this conclusion derisks the two-phase model.